src/Resources/config/routing/sonata_admin.xml

```diff
@@ -11,6 +11,6 @@
         <requirement key="_format">html|json</requirement>
     </route>
     <route id="sonata_admin_set_object_field_value" path="/core/set-object-field-value" controller="sonata.admin.action.set_object_field_value"/>
-    <route id="sonata_admin_search" path="/search" controller="Sonata\AdminBundle\Action\SearchAction"/>
+    <route id="sonata_admin_search" path="/search" controller="sonata.admin.action.search"/>
```

Comment:
Should not `sonata.admin.action.search` reference be added in `3.x`?

Reply:
https://github.com/sonata-project/SonataAdminBundle/pull/6589#issuecomment-727220777

Reply:
I didn't know. Thanks for providing the context.
Regardless the issue is not present in `3.x`, should not we make this change in `3.x` and then propagate it to `master`?

Reply:
You mean changing this ? 
https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Resources/config/actions.php#L39

It would be a BC-break no ?

Reply:
Yes, that's what I mean. if `Sonata\AdminBundle\Action\SearchAction` is still an alias, I think there is no BC break.

Reply:
Is https://github.com/sonata-project/SonataAdminBundle/pull/6622 ok ?